forge-gui/res/cardsfolder/upcoming/wrathful_raptors.txt

```diff
@@ -0,0 +1,10 @@
+Name:Wrathful Raptors
+ManaCost:4 R
+Types:Creature Dinosaur
+PT:5/5
+K:Trample
+T:Mode$ DamageDone | Execute$ TrigDamage | ValidTarget$ Dinosaur.YouCtrl | TriggerZones$ Battlefield | TriggerDescription$ Whenever a Dinosaur you control is dealt damage, it deals that much damage to any target that isn't a Dinosaur.
+SVar:TrigDamage:DB$ DealDamage | NumDmg$ X | ValidTgts$ Any.nonDinosaur | TgtPrompt$ Select any target that isn't a Dinosaur
```

Comment:
don't think it can target Players like that?

Reply:
I think it can (copied this from Wrathful Red Dragon) 

![image](https://github.com/Card-Forge/forge/assets/67333662/39aa1503-7378-43a2-9660-ac91d572146d)

Reply:
ok good